src/common/harvest/harvest.js

```diff
@@ -208,27 +204,16 @@ export function getSubmitMethod(endpoint, opts) {
   var useBody
 
   if (opts.needResponse) {  // currently: only STN needs a response
-    if (xhrUsable) {
-      useBody = true
-      method = submitData.xhr
-    } else {
-      return false
-    }
+    useBody = true
+    method = submitData.xhr
   } else if (opts.unload && isBrowserScope) { // all the features' final harvest; neither methods work outside window context
     useBody = haveSendBeacon
     method = haveSendBeacon ? submitData.beacon : submitData.img  // really only IE doesn't have Beacon API for web browsers
   } else {
     // `submitData.beacon` was removed, there is an upper limit to the
-    // number of data allowed before it starts failing, so we save it for
-    // unload data
-    if (xhrUsable) {  // this is practically every browser-version in use today, including all workers
-      useBody = true
-      method = submitData.xhr
-    } else if (endpoint === 'events' || endpoint === 'jserrors') {
-      method = submitData.img
-    } else {
-      return false
-    }
```

Comment:
Just a general question....with this change, does anything still use the img method?

Reply:
Looks like:

- IE on unload
- if harvest using sendBeacon fails
- by `inlineHit` api

So.. yes.